example/android/app/build.gradle

```diff
@@ -44,8 +44,8 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.example.example"
-        minSdkVersion 16
-        targetSdkVersion 30
+        minSdkVersion 30
+        targetSdkVersion 20
```

Comment:
Can not find any reason for changing the minSdkVersion 30 and targetSdkVersion 20

Reply:
Sorry, I have some configuration issue that day. Fixed in latest version.